packages/ui/components/icon/icon-list.mjs

```diff
@@ -0,0 +1,136 @@
+// icon name from lucide-react
+// this list is extracted based on what icons are currently used
+export const lucideIconList = new Set([
+  "x",
```

Comment:
list of icon names used throughout the app

Reply:
if we want to add a new icon, we have to add it here?